plasmapy/examples/plot_physics.py

```diff
@@ -57,4 +57,79 @@
     formulary.plasma_frequency(electron_concentration))
 plt.xlabel("Electron Concentration (m^-3)")
 plt.ylabel("Langmuir Wave Plasma Frequency (rad/s)")
+plt.grid()
+plt.show()
+
+############################################################
+# Let's compare our data to real-world data of plasma devices
+
+# Our 'Data'
+print("---Our Data---")
+print("n_e: " + str(electron_concentration))
+print("T_e: " + str(electron_temperature))
+print("B: " + str(B))
+plt.scatter(electron_concentration, formulary.plasma_frequency(electron_concentration))
+
+# IRT1 Tokamak Data
+# http://article.sapub.org/pdf/10.5923.j.jnpp.20110101.03.pdf
+n_e = 1.2e19 / u.m**3
+T_e = 136.8323 * u.eV
+B = 0.82 * u.T
+print("---IRT1 Tokamak Data---")
+print("n_e: " + str(n_e))
+print("T_e: " + str(T_e))
+print("B: " + str(B))
+plt.scatter(n_e, formulary.plasma_frequency(n_e))
+
+# Wendelstein 7-X Stellerator Data
+# https://nucleus.iaea.org/sites/fusionportal/Shared%20Documents/FEC%202016/fec2016-preprints/preprint0541.pdf
+n_e = 3e19 / u.m**3
+T_e = 6 * u.keV
+B = 3 * u.T
+print("---Wendelstein 7-X Stellerator Data---")
+print("n_e: " + str(n_e))
+print("T_e: " + str(T_e))
+print("B: " + str(B))
+plt.scatter(n_e, formulary.plasma_frequency(n_e))
```

Comment:
Does it work if you define labels at each (scatter)plot, so you don't have override them later in `plt.legend`?
```suggestion
plt.scatter(n_e, formulary.plasma_frequency(n_e), label="Wendelstein 7-X") 
```